src/adapter/input/post_router.py

```
@@ -82,6 +100,25 @@ def get_post_detail(post_id:str, service: PostApplicationService = Depends(get_p
+        raise HTTPException(
+            status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
+            detail=f"Error in request process: {str(e)}") from e
+
+@router.post(path='/{post_id}/likes',
+            status_code=status.HTTP_200_OK,
+            responses={
+                404: {"description": "Post Not Found"},
+                429: {"description": "Too Many Request"},
+                500: {"description": "Internal Server Error"}
+            })
+def add_like_post(post_id:str, service: PostApplicationService = Depends(get_post_application_service)):
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)